x-pack/plugins/observability_log_explorer/public/state_machines/origin_interpreter/src/notifications.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IToasts } from '@kbn/core-notifications-browser';
+import { mountReactNode } from '@kbn/core-mount-utils-browser-internal';
+import { EuiButton, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { LOGS_ONBOARDING_FEEDBACK_LINK } from '@kbn/observability-shared-plugin/common';
+import React from 'react';
+
+export const createRequestFeedbackNotifier = (toasts: IToasts) => () => {
+  toasts.addInfo({
```

Comment:
By design definition, the icon should be the comment one, you can set it using the iconType property for the toast:
```ts
iconType: 'editorComment'
```

Reply:
👍 Good catch. I forgot to add this on.